lib/simulator-xcode-6.js

```diff
@@ -919,7 +920,7 @@ class SimulatorXcode6 extends EventEmitter {
    *
    * @param {boolean} isEnabled - Set it to false in order to uncheck 'Toggle Enrolled State' flag
    */
-  async enrollTouchID (isEnabled = true) {
+  async enrollBiometric (isEnabled = true) {
     await this.executeUIClientScript(`
       tell application "System Events"
         tell process "Simulator"
```

Comment:
Should also menu item names be updated in the script?

Reply:
Those are updated in the xcode-9 simulator.... for xcode-6 it just assumes touchId